BSPConversionLib/Source/EntityConverter.cs

```diff
@@ -187,14 +195,20 @@ private static void OpenDoorOnPressed(Entity button, Entity door)
 
 		private static void SetButtonFlags(Entity button)
 		{
+			var spawnflags = 0;
```

Comment:
Move this variable to line 202. It is only used if `float.TryParse` returns true.